File: Python/TextConverter/unicode_to_html_converter.py
import cgi

class UnicodeFileToHtmlTextConverter(object):

    def __init__(self, full_filename_with_path):
        self.full_filename_with_path = full_filename_with_path

    def convert_to_html(self):
        f = open(self.full_filename_with_path, "r")
        html = ""
        for line in f:
            line = line.rstrip()
            html += cgi.escape(line, quote=True)
            html += "<br />"

        return html

File: Ruby/TextConverter/unicode_to_html_converter.rb
require 'cgi'

class UnicodeFileToHtmlTextConverter
  attr_accessor :full_filename_with_path
  def initialize(full_filename_with_path)
    @full_filename_with_path = full_filename_with_path
  end

  def convert_to_html
    html = ""
    File.open(self.full_filename_with_path, "r") do |f|
      while (line = f.gets)
          line = line.chomp()
          html += CGI.escapeHTML(line)
          html += "<br />"
      end
    end

    return html
  end
end
